src/externalsystems/OfferProvider.Library/BusinessLogic/OfferProviderBusinessLogic.cs

```diff
@@ -121,35 +121,38 @@ await _offerProviderService
             throw new ConflictException("Client should be set");
         }
 
-        if (data.ServiceAccounts.Count() > 1)
-        {
-            throw new ConflictException("There should be not more than one service account for the offer subscription");
-        }
+        List<CallbackTechnicalUserInfoData>? technicalUsersInfoData = null;
```

Comment:
doing a data-transformation by foreach... add to list... is so 90th. Today there's LINQ for that
```suggestion
        IEnumerable<CallbackTechnicalUserInfoData>? technicalUsersInfoData = null;
```